common.c

```diff
@@ -3438,7 +3438,8 @@ ifaddrconf(cmd, ifname, addr, plen, pltime, vltime)
 	memset(&req, 0, sizeof(req));
 #ifdef __KAME__
 	req.ifra_addr = *addr;
-	memcpy(req.ifra_name, ifname, sizeof(req.ifra_name));
+	memset(req.ifra_name, 0, sizeof(req.ifra_name));
+	strncpy(req.ifra_name, ifname, IFNAMSIZ - 1);
```

Comment:
```suggestion
	strlcpy(req.ifra_name, ifname, sizeof(req.ifra_name));
```

Memset was already done above for the whole struct and the ifconfig code in FreeBSD does this instead.. I think this is just about passing the end of the string here and we can avoid the magic constants and use the struct (future-proof, if ever, FreeBSD IFNAMSIZ is clamped at 16 bytes still).